pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -99,12 +102,13 @@ func FromMetrics(md pmetric.Metrics, settings Settings) (tsMap map[string]*promp
 						errs = multierr.Append(errs, fmt.Errorf("empty data points. %s is dropped", metric.Name()))
 					}
 					for x := 0; x < dataPoints.Len(); x++ {
-						addSingleSummaryDataPoint(dataPoints.At(x), resource, metric, settings, tsMap)
+						addSingleSummaryDataPoint(dataPoints.At(x), resource, scope, metric, settings, tsMap)
 					}
 				default:
 					errs = multierr.Append(errs, errors.New("unsupported metric type"))
 				}
 			}
+			addScopeTargetInfo(scope, resource, settings, mostRecentTimestamp, tsMap)
```

Comment:
nit, make the signature respect the hierarchy: `addScopeTargetInfo(resource, scope, settings, mostRecentTimestamp, tsMap)`